main.go

```diff
@@ -79,9 +80,32 @@ func main() {
 		exit(3)
 	}
 
+	// validate alternate entrypoints
+	entrypoints := strings.Split(*altEntrypointsFlag, ",")
+	grammar := g.(*ast.Grammar)
+	for _, entrypoint := range entrypoints {
+		if entrypoint == "" {
+			continue
+		}
+
+		found := false
+		// TODO(mna): build set of rule names
```

Comment:
This can be done easily. After line 85 (`grammar := g.(*ast.Grammar)`) insert:
```
rules := map[string]struct{}{}
for _, rule := grammar.Rules {
    rules[rule] = struct{}{}
}
```

And then perform the check like this:
```
if _, ok := rules[rule]; !ok {
    fmt.Fprintf(os.Stderr, "argument error:\nunknown rule name %s used as alternate entrypoint\n", entrypoint)
    exit(9)
}
```

Reply:
:) yeah I know, it was just laziness in the first draft, that's why I added the comment to get back to it later.